### PR TITLE
Mimir 456 front page banner adjustments

### PR DESCRIPTION
**Pull request by itemodahl**

### CODE REVIEW

File: src/main/resources/lib/ssb/keyFigure.ts [hunk 1]
```diff
@@ -191,7 +191,7 @@ function getDataTbProcessor(
 
   const tr: Array<TableCellUniform> = head[head.length - 1].tr
   const th: Array<number | string | PreliminaryData> = tr[head.length - 1].th
-  keyFigureViewData.time = th[0].toString()
+  keyFigureViewData.time = Array.isArray(th[0]) ? th[0].join(' ').toString() : th[0].toString()
```
Comment (johnnadeluy) [inline src/main/resources/lib/ssb/keyFigure.ts:194]: ?

File: src/main/resources/site/parts/frontPageBanner/frontPageBanner.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <section class="xp-part part-frontpage-banner">
     <div class="container">
         <div class="row">
-            <div class="col-12 col-lg-6 mx-auto mb-4">
+            <div class="col-12 col-lg-6 mx-auto mb-md-2 mt-2 text-lg-left text-center">
                 <div class="d-inline-block banner-text-wrapper">
```
Comment (johnnadeluy) [inline src/main/resources/site/parts/frontPageBanner/frontPageBanner.html:5]: Margin top is a bit off on mobile for me still. Might have something to do with the banner-text-wrapper class.

**Review by johnnadeluy — CHANGES_REQUESTED** [review]

File: src/main/resources/lib/ssb/keyFigure.ts (hunk 1 above)
Comment (itemodahl) [reply src/main/resources/lib/ssb/keyFigure.ts:194]: Jeg lagde en ny branch fra en annen branch en master. Denne snutten over er nå allerede merget inn.

File: src/main/resources/site/parts/frontPageBanner/frontPageBanner.html (hunk 2 above)
Comment (itemodahl) [reply src/main/resources/site/parts/frontPageBanner/frontPageBanner.html:5]: Changed it to use flex instead.

**Review by johnnadeluy — APPROVED** [review]
Looks good! Well done 🙌

**Pull request merged**